### DISCUSSION TITLE
Estimated UR statistic for scores without replays

### DISCUSSION THREAD

**Discussion by Natelytle** [ppy/osu#26420] [Ideas]
Since there are 3 pull requests open adding unstable rate estimation each mode (#22613, #20963, #15091), I was thinking that a neat little metric could be to display the estimated unstable rate in place of the true unstable rate when a score hasn't been run or has no replay. The estimation isn't 100% accurate but its pretty close for most scores.

An example implementation on mania (also showcasing how robust it is, with 4400 misses):
![image](https://github.com/ppy/osu/assets/92956514/f3dd199c-ab35-4e01-b99a-9e32aa476122)

**Comment by GRBL-F** [comment]
i'd like to see this along with the hitcircle heatmap to see how you're struggling so you know how to adjust your aim too. especially for farmers like myself who reset often mid map.

**Comment by hostilew** [comment]
would love to see this implemented if the new results screen doesn't auto fetch replay statistics. Browsing local scores and having to watch every replay to get a UR is a bit tedious.
